voicer-client/src/components/audio/AudioUploader.jsx

```diff
@@ -25,9 +30,11 @@ const ReactUploadMedia = () => {
 		axiosWithAuth()
 		.post("/api/voice", formData, config)
 			.then(res => {
-				alert("File successfully uploaded")
+				refreshAppHandler()
+				// alert("File successfully uploaded")
```

Comment:
smart, brought the refresh so the page will show the new item

Reply:
Clean Code!